Library/Homebrew/dev-cmd/bump-cask-pr.rb

```diff
@@ -174,6 +170,15 @@ def bump_cask_pr
     GitHub.create_bump_pr(pr_info, args: args)
   end
 
+  sig { params(version: Cask::DSL::Version, cask: Cask::Cask).returns(Cask::DSL::Version) }
+  def shortened_version(version, cask:)
+    if version.before_comma == cask.version.before_comma
```

Comment:
```suggestion
    if version.before_comma == cask.version
```

Reply:
That doesn't sound like it would ever be equal unless you are removing the comma in the bump.

The fix here was simply to share the existing logic used to title the commit/PR with the search functionality. They were previously not aligned so the search would of course not find the differently titled PR.

Reply:
I was seeing the same issue where it wasn't working now, but should moving forward as your comment below implies.

Reply:
Aren't we now returning the whole version as the shortened version of the `before_comma` matches for the cask and the bump? Shouldn't we always return the before_comma?